src/metabase/search/appdb/core.clj

```diff
@@ -31,7 +35,7 @@
 
 (def supported-db?
   "All the databases which we have implemented fulltext search for."
-  #{:postgres :h2})
+  #{:postgres :h2 :mysql})
```

Comment:
Since the implementation at this point yields an inferior customer experience to old search, we need to be careful that it doesn't become the default for MySQL customers when we do so for Postgres.

I'm not sure the best solution for this, but off the top of my head I have one idea - use a different keyword for the engine and derive it from `:search.engine/appdb`.